src/plugins/GenerateJob/templates/backend_deepforge.py

```diff
@@ -301,6 +306,27 @@ def figure_to_state(self):
             state['axes'].append(axes_data)
         return state
 
+    def process_line_collection(self, collection):
+        line_collections = []
+        colors = collection.get_colors()
+        ls = collection.get_dashes()
+        lw = collection.get_linewidths()
+
+        for i, segment  in enumerate(collection.get_segments()):
+            line_collection_data = dict()
+            line_collection_data['points'] = segment.tolist()
+            label = collection.get_label()
+            if label is None:
+                line_collection_data['label'] = ''
+            else:
+                line_collection_data['label'] = label
```

Comment:
Why can't this just be `line_collection_data['label'] = label`?

Reply:
I am thinking of removing this label regardless. Because if there are 50 linesegments in the  collection, it will clutter the whole plot with `_collection0`.